src/sql/workbench/contrib/notebook/browser/cellViews/textCell.css

```diff
@@ -15,25 +15,43 @@ text-cell-component code-component {
 }
 text-cell-component .notebook-preview {
 	flex-direction: column;
-	width: 100%;
+	padding-left: 8px;
+	padding-right: 8px;
 	user-select: none;
+	width: 100%;
+}
+text-cell-component .text-view {
+	flex-direction: column;
 	padding-left: 8px;
 	padding-right: 8px;
+	user-select: none;
+	width: 100%;
 }
 text-cell-component .edit-mode code-component {
 	display: block;
 	width: 100%;
 }
 text-cell-component .show-preview.edit-mode code-component {
-	width: 50%;
+	height: 1px;
+	width: 1px;
 }
 text-cell-component .edit-mode .notebook-preview {
 	border-color: transparent;
 	border-left-width: 1px;
 	border-style: solid;
 	border-top-width: 0;
-	flex-direction: column;
-	width: 50%;
+	/* hiding for the moment */
```

Comment:
What's the intention here? Is this just for testing or something?

Reply:
This was for testing b/c we already had preview showing by default. I'll clean that up.